src/App.jsx

```diff
@@ -14,33 +15,87 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABETICAL = 'alphabetical';
+const SORT_BY_LENGHT = 'length';
```

Comment:
There is a typo in the constant name `SORT_BY_LENGHT`. It should be `SORT_BY_LENGTH` to correctly reflect the word 'length'.